src/test/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinBuilderJvmAbiTest.java

```diff
@@ -38,6 +38,7 @@ public void testGeneratesAbiOnly() {
           c.compileKotlin();
           c.outputJdeps();
           c.outputJavaJdeps();
+          c.outputJar();
```

Comment:
Without this, the skip-code-gen plugin gets added, which causes the new ABI compiler to short circuit and not output anything.